test/test_forcings.jl

```diff
@@ -119,19 +119,28 @@ function advective_and_multiple_forcing(arch)
     grid = RectilinearGrid(arch, size=(2, 2, 3), extent=(1, 1, 1), halo=(4, 4, 4))
 
     constant_slip = AdvectiveForcing(w=1)
-
     no_penetration = ImpenetrableBoundaryCondition()
     slip_bcs = FieldBoundaryConditions(grid, (Center, Center, Face), top=no_penetration, bottom=no_penetration)
     slip_velocity = ZFaceField(grid, boundary_conditions=slip_bcs)
+    set!(slip_velocity, 1)
     velocity_field_slip = AdvectiveForcing(w=slip_velocity)
-    simple_forcing(x, y, z, t) = 1
+    simple_forcing(x, y, z, t) = 0
 
     model = NonhydrostaticModel(; grid, tracers=(:a, :b), forcing=(a=constant_slip, b=(simple_forcing, velocity_field_slip)))
+
+    a₀ = rand(size(grid)...)
+    b₀ = rand(size(grid)...)
+    set!(model, a=a₀, b=b₀)
+
+    # Time-step without an error?
     time_step!(model, 1, euler=true)
 
-    return true
-end
+    # Does `b` change?
+    a₁ = Array(interior(model.tracers.a))
+    b₁ = Array(interior(model.tracers.b))
 
+    return a₁ ≠ a₀ && b₁ ≠ b₀
```

Comment:
In this test do we maybe want to somehow check that both of the forcings are being done, perhaps by a two step thing like doing this test, then changing `simple_forcing` to return 1 and setting the slip velocity to 0 and testing again?

Reply:
We can add a third tracer